L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -1050,6 +1050,11 @@ namespace trklet {
     //Rinv bin edges for 6 bins.
     std::vector<double> rinvBins_{-rinvcut(), -0.004968, -0.003828, 0, 0.003828, 0.004968, rinvcut()};
     //Phi bin edges for 2 bins.
+    //The bin edges for rinv were chosen using the output tracks without duplicate removal for a large number of events
```

Comment:
Can this very long comment be simplified to ine sentence "(rinv bins were optimised to ensure a similar number of tracks in each bin prior to DR)", without losing significant info?

Reply:
I've done so.